R/qenv-eval_code.R

```diff
@@ -37,7 +37,7 @@ setMethod("eval_code", signature = c("qenv", "character"), function(object, code
 
   for (i in seq_along(code_split)) {
     current_code <- code_split[[i]]
-    current_call <- parse(text = current_code, keep.source = FALSE)
+    current_call <- parse(text = trimws(current_code), keep.source = TRUE)
```

Comment:
Just curious, why white spaces are problematic?

Reply:
I think that's a remaining of when we also removed `{` (brackets) at the beginning and the end of the call. So we first removed whitespaces, and then brackets. I'll try to see if this `trimws` is still needed here. Probably not

Reply:
Removed! 8d3649e6b676dd9c1e4c2dc1016138c13d1be557